examples/caption_generation.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>Wrap Up section is not in the same size as the other section headers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/visual-layer/fastdup/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Thanks, I wanted all the code to be under one section, and the wrap-up to be under a different section, so that if the code is collapsed, the wrap up section will still be displayed.